docs/reference/health/health.asciidoc

```diff
@@ -365,6 +365,43 @@ watermark threshold>>.
 `policies`::
     (int) The number of index lifecycle policies that the system is managing.
 
+`stagnating_indices`::
+    (int) the number of indices –managed by an Index Lifecycle Policy– that has been stagnant longer than expected.
```

Comment:
I would recommend to use the macros, either `{ilm}` or `{ilm-init}` if it has been mentioned before, and rephrase it to `managed by ILM`